pyhiveapi/apyhiveapi/session.py

```diff
@@ -101,6 +100,8 @@ def __init__(
         )
         self.devices = {}
         self.deviceList = {}
+        self.api = API(hiveSession=self, websession=websession)
```

Comment:
Not sure why this has moved? don't really matter I don't think the linter may complain

Reply:
iirc there's a race condition here. this makes sure that the api is not defined until after session has everything it needs